app/controllers/PropertyDetailsValuesController.scala

```diff
@@ -81,32 +82,82 @@ trait PropertyDetailsValuesController extends BackendController  {
       withJsonBody[PropertyDetailsOwnedBefore] { draftPropertyDetails =>
         propertyDetailsService.cacheDraftPropertyDetailsOwnedBefore(atedRefNo, id, draftPropertyDetails).map { updatedDraftPropertyDetails =>
           updatedDraftPropertyDetails match {
-            case Some(x) => Ok("")
+            case Some(_) => Ok("")
             case None => BadRequest("Invalid Request")
           }
         }
       }
   }
 
-  def saveDraftPropertyDetailsNewBuild(atedRefNo: String, id: String) = Action.async(parse.json) {
+  //TODO refactor/remove the method below in a future ticket
```

Comment:
What is the reason that this is to be removed in a future ticket?

Reply:
Removed